Sources/NIOCore/EventLoop.swift

```diff
@@ -781,6 +781,31 @@ extension EventLoop {
     ) -> Scheduled<T> {
         self._flatScheduleTask(in: delay, file: file, line: line, task)
     }
+    
+    /// Schedule a `task` that is executed by this `EventLoop` after the given amount of time.
+    ///
+    /// - parameters:
+    ///     - delay: The delay between the end of one task and the start of the next.
+    ///     - maximumAllowableJitter: Exclusive upper bound of jitter range added to the `delay` parameter.
+    ///     - task: The asynchronous task to run. As everything that runs on the `EventLoop`, it must not block.
+    /// - returns: A `Scheduled` object which may be used to cancel the task if it has not yet run, or to wait
+    ///            on the full execution of the task, including its returned `EventLoopFuture`.
+    ///
+    /// - note: You can only cancel a task before it has started executing.
+    @discardableResult
+    @inlinable
+    @preconcurrency
+    public func flatScheduleTask<T>(
```

Comment:
Can we also add an extension that offers a variation of `scheduleTask` with `jitter`?

Reply:
A function that schedules a task after a given amount of time, with that amount being jittered? If this is the case I don't see the difference with this overloaded function, and as you pointed out as there's no rescheduling jittering should be unnecessary.